### ISSUE TITLE
Tensorflow 2.0 support

### ISSUE THREAD

**Issue by mankeyboy** [NVIDIA/DeepLearningExamples#201]
Hi, is there any planned timeline for the Tensorflow codebases to support TF2.0 API in the NGC containers and in this repository?

**Comment by nvpstr** [comment]
First models for TF2 should be released in December or January. Stay tuned.

**Comment by pstjohn** [comment]
Any updates on when TF2 models might be released? I'm especially interested in a TF2 model with gradient accumulation, as was done in the TF1 BERT model.
